src/prefect/server/models/flows.py

```diff
@@ -125,13 +129,13 @@ async def read_flow_by_name(
 @db_injector
 async def _apply_flow_filters(
     db: PrefectDBInterface,
-    query,
-    flow_filter: schemas.filters.FlowFilter = None,
-    flow_run_filter: schemas.filters.FlowRunFilter = None,
-    task_run_filter: schemas.filters.TaskRunFilter = None,
-    deployment_filter: schemas.filters.DeploymentFilter = None,
-    work_pool_filter: schemas.filters.WorkPoolFilter = None,
-):
+    query: Select[T],
+    flow_filter: Union[schemas.filters.FlowFilter, None] = None,
```

Comment:
Not `Optional`?

Reply:
I dunno, I like this way better because it's closer in spirit to `Thing | None`, but happy to go straight Optional.